### PR TITLE
Streaming always use anonymous http connection

### PR DESCRIPTION
**Pull request by cvigo**
Streaming always use anonymous http connection, regardless of the "Use https" setting

Replaces PR #50

### CODE REVIEW

**Comment by Jalle19** [comment]
Can you also update the change log and the version number?

**Comment by cvigo** [comment]
change log and version number done

**Comment by cvigo** [comment]
Travis build keeps failing due to wrong references to xbmc project in `cmake`

**Comment by cvigo** [comment]
Travis fix committed

**Comment by Jalle19** [comment]
Can you rebase?

**Comment by cvigo** [comment]
Merged with master

**Comment by Jalle19** [comment]
You need to pull the latest master locally, then checkout this branch and run `git rebase master`, then force push this branch to GitHub.

**Comment by cvigo** [comment]
I think it's done now... pulled the latest kodi-pvr/pvr.vuplus master and merged into my Krypton

**Comment by Jalle19** [comment]
You can't merge master into your Krypton branch since the rest of the changes are not compatible. I suggest you focus on getting the change merged to master first, then you can make a new PR that targets Krypton. You basically had it in https://github.com/kodi-pvr/pvr.vuplus/pull/50 already, just rebase that branch on the latest master and force push to update it on GitHub.

**Comment by cvigo** [comment]
What a mess for a change of two C++ code...

I' start from the scratch, applying the changes to kripton and then to master

**Comment by cvigo** [comment]
Rebased Transifex latest 3 commits from the kodi-pvr/pvr.vuplus Krypton branch

**Comment by cvigo** [comment]
Rebased latest commits from kodi-pvr/pvr.vuplus Krypton to resolve PR conflicts

File: pvr.vuplus/changelog.txt [hunk 1]
```diff
@@ -1,3 +1,12 @@
+v2.4.7
```
Comment (Jalle19) [inline pvr.vuplus/changelog.txt]: `addon.xml.in` says the version is `2.4.9`, do you think you could change it here too so they match?

**Comment by cvigo** [comment]
I can... Version log is quite inconsistent across commits. Some update it, some do not (specially translation commits)

File: pvr.vuplus/changelog.txt (hunk 1 above)
Comment (cvigo) [inline pvr.vuplus/changelog.txt]: I can... Version log is quite inconsistent across commits. Some update it, some do not (specially translation commits)

Comment (cvigo) [inline pvr.vuplus/changelog.txt]: done

File: .travis.yml [hunk 2]
```diff
@@ -38,6 +38,6 @@ before_script:
   - cd $TRAVIS_BUILD_DIR/..
   - git clone --depth=1 https://github.com/xbmc/xbmc.git
   - cd pvr.vuplus && mkdir build && cd build
-  - cmake -DADDONS_TO_BUILD=pvr.vuplus -DADDON_SRC_PREFIX=$TRAVIS_BUILD_DIR/.. -DCMAKE_BUILD_TYPE=Debug -DCMAKE_INSTALL_PREFIX=$TRAVIS_BUILD_DIR/../xbmc/addons -DPACKAGE_ZIP=1 $TRAVIS_BUILD_DIR/../xbmc/project/cmake/addons
+  - cmake -DADDONS_TO_BUILD=pvr.vuplus -DADDON_SRC_PREFIX=$TRAVIS_BUILD_DIR/.. -DCMAKE_BUILD_TYPE=Debug -DCMAKE_INSTALL_PREFIX=$TRAVIS_BUILD_DIR/../xbmc/addons -DPACKAGE_ZIP=1 $TRAVIS_BUILD_DIR/../xbmc/cmake/addons
```
Comment (Jalle19) [inline .travis.yml]: Actually this is not right for the Krypton branch. The correct solution is to check out the `Krypton` branch of Kodi two lines above this one, then the path will be correct.

Comment (Jalle19) [inline .travis.yml]: https://github.com/kodi-pvr/pvr.hts/blob/Krypton/.travis.yml#L44 there's an example

Comment (cvigo) [inline .travis.yml]: yes that's right.
Fixed

**Comment by Jalle19** [comment]
Thanks! Can you make a corresponding PR against the master branch too?

**Pull request merged**